apps/frontend/src/app/5_pages/MarketMakingPage/components/AmbientMarketMaking/AmbientMarketMaking.constants.ts

```diff
@@ -0,0 +1,7 @@
+import { BOB_CHAIN_ID } from '../../../../../config/chains';
+
+import { AmbientLiquidityPool } from './utils/AmbientLiquidityPool';
+
+export const TESTNET_AMM = [
+  new AmbientLiquidityPool('dllr', 'rbtc', BOB_CHAIN_ID),
```

Comment:
These are available pools atm:

```suggestion
  new AmbientLiquidityPool('ETH', 'SOV', BOB_CHAIN_ID),
  new AmbientLiquidityPool('ETH', 'GLD', BOB_CHAIN_ID),
```